hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-soak.yaml

```diff
@@ -145,6 +145,7 @@
             post-env: ''
             soak-deploy: ''
             soak-continuous: ''
+            cron-string: 'H */6 * * *'
```

Comment:
The goal was to save executors as well as $. Do we need to run a 1.1 soak cluster at all?